scipy/stats/_rcont/_rcont.c

```diff
@@ -10,11 +10,11 @@
 
   Author: Hans Dembinski
 */
-
+#include <math.h>
```

Comment:
I thought at first that this was the wrong include order - (C standard library headers before `Python.h`). However, `Python.h` is not included anymore. It'd be useful to note in a comment (or `libnpyrandom/README.md`?) that none of that code should be depending on NumPy or Python.

Reply:
That's true. I'll add a note to `libnpyrandom/README.md`.